### ISSUE TITLE
"Skip to search" link doesn't work if you only have one search_field

### ISSUE THREAD

**Issue by jcoyne** [projectblacklight/blacklight#3302]
If in the configuration you only call `add_search_field` once, then the `search_field` form component is hidden.  However the "Skip to search" link still points at that hidden element.

See https://github.com/geoblacklight/geoblacklight/issues/1580
